docs/cmdlets-reference/new-cohesityhypervprotectionjob.md

```diff
@@ -0,0 +1,90 @@
+# New-CohesityHypervProtectionJob
+
+## SYNOPSIS
+Create a new protection job for HyperV source.
+
+## DESCRIPTION
+Returns the created protection job for HyperV source.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+New-CohesityHypervProtectionJob -Name <string> -PolicyName <string> -StorageDomainName <string> -SourceName <string>
```

Comment:
Can we give concrete examples ? customers do this from get-help .

Reply:
Added a sample example